### ISSUE TITLE
[a11y-defect-3]: YouTube iframe embeds are larger than VA minimum breakpoint

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#64211]
### Point of contact

Jamie Klenetsky Fay @coforma-jamie

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

In the VR&E Orientation slideshow, there are a few slides with embedded YouTube videos.

Those videos' width are set, inline, as 325px. The [VA's minimum breakpoint]
(https://design.va.gov/foundation/breakpoints) is 320px. At 320px, the video overflows off of the screen.  This is an especially big issue for users that zoom in to 400%. Those users would need to scroll horizontally to see the full video. While it's only off by a few pixels, something could still be cut off for these users.

![Screenshot 2023-08-17 at 4 34 46 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/5433113f-af24-4f0b-9470-3e1b11635033)
]

### Reproduction steps

1. Go to [the form](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)
2. In the Wizard component, select "Veteran > Yes > Yes"
3. Scroll down to see the "VR&E Orientation" slideshow
4. Click the "Start VR&E orientation slideshow" button
5. Click "Next slide" until you get to slide 5, where you'll see an embedded YouTube video
6. Resize your browser to 1280px, then zoom in to 400%

### Proposed solution or next steps

At the time of this writing, the design system doesn't have any classes to handle 16:9 ratios. We can give you two options to address this issue in the meantime:

### Link to the videos instead
You can use a design system [video link](https://design.va.gov/components/link/#video) to link out to each of the videos instead. Since it would interrupt the user's flow to go to a new page to view the video(s), you'll need to note that it opens in a new tab: "Go to the video about X on YouTube (opens in new tab)".

Pros of this approach: 
- Your pages will load faster without the video embeds
- You won't have to worry about any accessibility issues caused by the third-party (YouTube) embed 
- It solves the accessibility issue

Cons of this approach:
- You'll lose the utility of the video being on the same page as the rest of the slideshow
- Folks will have to go to a new tab to view each video

### Use utility classes to make the video responsive
You can
1. remove the height and width from each embed
2. add [height and width utility classes](https://design.va.gov/foundation/utilities/height-and-width#responsive-prefixes) to make the embed auto-size with the container or viewport

Pros of this approach:
- You can keep the videos on the page, as is
- It solves the accessibility issue

Cons of this approach:
- There are no 16:9 ratio classes. **You will want to make this as a recommendation to the DST.** As a result, you'll need to play around with the different utility classes until you find something that works well enough...and it won't be in the proper 16:9 ratio.

### References, articles, or WCAG support

1. [Height and width utility classes](https://design.va.gov/foundation/utilities/height-and-width#responsive-prefixes)
2. [Video links](https://design.va.gov/components/link/#video)
3. [WCAG 1.4.10: Reflow](https://www.w3.org/TR/WCAG21/#reflow)

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [X] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [X] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
```[tasklist]
### Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/68025
```

**Comment by steele-lm** [comment]
@fiorella-io Do you have any preference for which approach for the video is implemented?

**Comment by fiorella-io** [comment]
@zack @steele-lm I'd prefer we keep the video on the page and try the second approach, but if it becomes too cumbersome, we can have the link.
